activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -561,7 +562,7 @@ def column_exists?(column_name, type = nil, options = {})
       #  t.index([:branch_id, :party_id], unique: true, name: 'by_branch_party')
       #
       # See {connection.add_index}[rdoc-ref:SchemaStatements#add_index] for details of the options you can use.
-      def index(column_name, options = {})
+      def index(column_name, options={})
```

Comment:
Are these changes (`options = {}` to `options={}`) automatically appried?
`Layout/SpaceAroundOperators` is temoprarily disabled in #36943 since that cop changed to enforce hard alignment too, but originally we prefer `options = {}` over `options={}`.

Reply:
It's my mistake I'll revert those befor easking for review.

This PR is still a WIP at this stage, sorry.